sdk/monitor/Azure.Monitory.Query/tests/LogsClientSamples.cs

```diff
@@ -74,8 +77,11 @@ public async Task QueryLogsAsPrimitive()
             #region Snippet:QueryLogsAsPrimitive
 
             LogsClient client = new LogsClient(new DefaultAzureCredential());
-            /*@@*/string workspaceId = TestEnvironment.WorkspaceId;
-            //@@string workspaceId = "<workspace_id>";
+#if SNIPPET
+            string workspaceId = "<workspace_id>";
+#else
```

Comment:
FWIW, in Key Vault there's many places we do something like this:

```csharp
KeyClient client = new KeyClient(...);
/*@@*/ client = GetClient(); // For recorded samples/tests
```

Which should work simply as:

```csharp
KeyClient client = new KeyClient(...);
#if !SNIPPET
client = GetcClient();
#endif
```

I suppose you could declare the local like this in either case as well.